libraries/standard/mqtt/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# Include filepaths for source and include.
+include(filePaths.cmake)
+
+# MQTT library target.
+add_library( mqtt
+             ${MQTT_SOURCES}
+             include/mqtt.h
```

Comment:
I know that `v4_beta` adds header files as "sources" in `add_library` which was probably for showing headers as source for IDE integration (Header path are otherwise only required to be included through `target_include_directories`).
It would be nice if it can be avoided in this new branch. However, I leave that upto @leegeth for suggestion from SSoT point of view.

Reply:
Right, I was wondering why v4_beta had header files as sources but just did the same to be safe.

Reply:
I agree to Archit's comment regarding removing header files from add_library